### DISCUSSION TITLE
横版冈易我的世界改版

### DISCUSSION THREAD

**Discussion by sb35386057** [share121/inter-knot#5257] [灌水]
![Screenshot_2024-09-21-10-00-37-197_com netease mc mi](https://github.com/user-attachments/assets/f1b1dc1c-1dad-4cc2-a91a-214e6d74a640)
这个就叫高玩派对，一模一样，还有就是更新了后，有一些模组出事了，过几天看看有能不能修回来
![68747470733a2f2f696d672d7374617469632e6d69686f796f2e636f6d2f636f6d6d756e6974797765622f75706c6f61642f31326138306138663564613339343962303836633439376238636334623564332e706e67](https://github.com/user-attachments/assets/6c99be41-b53a-4f8c-89d8-0e995e028678)

**Comment by cpu644** [comment]
这 这 这是个什么东西 wc 
![image](https://github.com/user-attachments/assets/6276ab80-737b-4c59-a8a1-124d914742cd)

**Reply by Kazuha-hhh** [reply]
味可太冲了

**Comment by Mitishirube0717** [comment]
什么派对？

**Reply by manyshoes** [reply]
尕佤派对
